pkg/console/subresource/consoleserver/types.go

```diff
@@ -22,9 +22,14 @@ type Config struct {
 	Auth                     `yaml:"auth"`
 	Customization            `yaml:"customization"`
 	Providers                `yaml:"providers"`
-	Plugins                  map[string]string `yaml:"plugins,omitempty"`
-	Proxy                    Proxy             `yaml:"proxy,omitempty"`
-	ManagedClusterConfigFile string            `yaml:"managedClusterConfigFile,omitempty"`
+	Plugins                  Plugins `yaml:"plugins,omitempty"`
+	Proxy                    Proxy   `yaml:"proxy,omitempty"`
+	ManagedClusterConfigFile string  `yaml:"managedClusterConfigFile,omitempty"`
+}
+
+type Plugins struct {
+	Enabled    map[string]string `yaml:"enabled,omitempty"`
```

Comment:
`enabled` sounds like it should be a boolean value. If we're going to break the API, I would change `plugins` to be an array of objects with values like `name`, `url`, `proxy`, `namespaces`, etc. I would be OK with leaving `plugins` as-is and just adding a separate `i18nNamespaces` array prop.